migrations/versions/10_adding_supplier_to_user_table.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: 10_adding_supplier_to_user_table
+Revises: 550715127385
+Create Date: 2015-04-21 07:45:29.958447
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '10_adding_supplier_to_user_table'
```

Comment:
This will actually be revision 20 rather than 10, so maybe the name of this file/Revision ID should reflect this.  Would be good to include this revision in the VERSION_HISTORY file too: https://github.com/alphagov/digitalmarketplace-api/blob/master/migrations/VERSION-HISTORY.txt